### ISSUE TITLE
hello,can you loading your resnet50 code for training?

### ISSUE THREAD

**Issue by Soulempty** [tzzcl/RNN-HA#2]
I use resnet50  to train,but there is some puzzle,accuracy is not right.
thanks you.

**Comment by tzzcl** [comment]
Hi, For resnet50 training, it follows the same process as in README.md.

But note that you should remove InvertNormalize() in both fine_tun.py and training test code.

I will upload resnet50 code after the ICCV deadline.

**Comment by Soulempty** [comment]
Thank you for your reply.I replace the InvertNormalize with transforms.Normalize(),but the loss always shake and the accuracy is not steady.
I upload the code to my RNN_HA repository,if you have free time,you can help me check the train_RnnHa.py and RNN_HA..py.
THANK YOU VERY MUCH.

**Comment by tzzcl** [comment]
Hi, I'm updated the repo for adding resnet training codes!

**Comment by Soulempty** [comment]
I know,thank you.
